examples/docker/modin-ray/Dockerfile

```diff
@@ -31,26 +35,37 @@ RUN adduser --disabled-password \
     --home $HOME \
     $USER
 
-ENV CONDA_DIR ${HOME}/miniconda
-
-SHELL ["/bin/bash", "--login", "-c"]
+# Conda settings
+ENV CONDA_DIR=${HOME}/miniconda
+ENV CONDA_ENV_NAME=modin-ray
+ENV PATH="${CONDA_DIR}/bin:${PATH}"
 
 RUN wget -nv https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O /tmp/miniconda3.sh && \
     bash /tmp/miniconda3.sh -b -p "${CONDA_DIR}" -f -u && \
     "${CONDA_DIR}/bin/conda" init bash && \
-    rm -f /tmp/miniconda3.sh && \
-    echo ". '${CONDA_DIR}/etc/profile.d/conda.sh'" >> "${HOME}/.profile"
+    rm -f /tmp/miniconda3.sh
 
 RUN conda update -n base -c defaults conda -y && \
-    conda config --set channel_priority strict && \
-    conda create -n modin --yes --no-default-packages && \
-    conda activate modin && \
-    conda install -c conda-forge modin-omnisci && \
+    conda create -n ${CONDA_ENV_NAME} --yes -c conda-forge modin-ray ray-dashboard scikit-learn scikit-learn-intelex xgboost && \
```

Comment:
you added `ray-dashboard` package, why?

Reply:
It removes Ray warning that it cannot find dashboard. It may be possible to monitor Ray state using dashboard web interface. On cnvrg.io platform Ray dashboard is integrated into its web interface.